src/App.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import { nanoid } from "nanoid";
+import Confetti from "react-confetti";
+
+function Die(props) {
+  const styles = {
+    backgroundColor: props.isHeld ? "#59E391" : "white",
+  };
+  return (
+    <div className="die-face" style={styles} onClick={props.holdDice}>
+      <h2 className="die-num">{props.value}</h2>
+    </div>
+  );
+}
```

Comment:
Can you use Destructuring in this project? It improves the sustainability, readability of code.
For more info check this link [here](https://www.geeksforgeeks.org/destructuring-of-props-in-reactjs/)